redhat-acm/resource-exhaustion/resourcequota-policy/resourcequota-policy.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-resourcequota
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: SC System and Communications Protection
+    policy.open-cluster-management.io/controls: SC-6 Resource Availability
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-resourcequota-example
+        spec:
+          remediationAction: inform # the policy-template spec.remediationAction is overridden by the preceding parameter value for spec.remediationAction.
+          severity: medium
+          namespaceSelector:
+            include: ["default"] # Namespace on which the policy will take affect
```

Comment:
@michaelkotelnikov @tommeramber  Thoughts?
I think it will be more practical to include ResourceQuota for all namespaces except : 
openshift*
open-*
kube-*

In that way we enforce resources quotas on all application projects in openshift cluster.

Reply:
The default namespace is a default templating value for the policy. Each policy consumer will edit the namespaceSelectors according to organizational standard.